src/main/java/com/juniorstart/juniorstart/payload/ChangePasswordRequest.java

```diff
@@ -0,0 +1,35 @@
+package com.juniorstart.juniorstart.payload;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.Max;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotBlank;
+import lombok.Getter;
+
+import java.util.UUID;
+
+/** Class for method changeEmail from UserService
+ * @author Dawid Wit
+ * @version 1.0
+ * @since 1.0
+ */
+@Getter
+public class ChangePasswordRequest {
+    @Email
+    String newPassword;
```

Comment:
Tutaj to samo :P private final I będzie gitarka

Reply:
private wystarczy